src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -155,12 +155,22 @@
     @Argument(doc = "Write the original contig/position for lifted variants to the INFO field.", optional = true)
     public boolean WRITE_ORIGINAL_POSITION = false;
 
+    // Option on whether or not to write the original alleles of the variant to the INFO field
+    @Argument(doc = "Write the original alleles for lifted variants to the INFO field.  If the alleles are identical, this attribute will be omitted.", optional = true)
+    public boolean WRITE_ORIGINAL_ALLELES = false;
+
     @Argument(doc = "The minimum percent match required for a variant to be lifted.", optional = true)
     public double LIFTOVER_MIN_MATCH = 1.0;
 
     @Argument(doc = "Allow INFO and FORMAT in the records that are not found in the header", optional = true)
     public boolean ALLOW_MISSING_FIELDS_IN_HEADER = false;
 
+
+    @Argument(doc = "If the REF allele of the lifted site does not match the target genome, that variant is normally rejected. " +
+            "For bi-allelic SNPs, if this is set to true and the ALT allele equals the new REF allele, the REF and ALT alleles will be swapped.  This can rescue " +
+            "some variants; however, do this carefully as some annotations may become invalid, such as any that are alelle-specifc.  See also TAGS_TO_REVERSE and TAGS_TO_DROP.", optional = true)
+    public boolean RECOVER_SWAPPED_REF_ALT = false;
```

Comment:
Out of curiosity, why are we changing the default behavior of this tool?  Why not keep it the way it currently is and add a DO_NOT_RECOVER_SWAPPED_REF_ALT option instead?

Reply:
Like I noted in the description of the PR, I figured this might need some discussion.  My reasoning is in #1149.  The main reason I believe the current default is a bad idea is b/c of the high likelihood of leaving the VCF in a bad/inaccurate state with respect to annotations.  As a general rule, I think tools like picard should take the more conservative default position.  This seems like a reasonable feature a user might want in some cases, hence the additional logging I added in the PR around features that could be rescued if this was turned on.  However, that user needs to think about the annotations specific to their VCF and the consequences of this swap before blindly doing it.

Reply:
Okay, fair enough.  I would just want to hear from @yfarjoun to make sure he does not object (since he made the original changes).